pages/discover.js

```diff
@@ -20,62 +20,61 @@ export default function Discover() {
       <Title content="What is Condensation?" title="A system made for data trust and ownership." />
       <SubTitle
         content="Overview"
-        title="CDS is an industry oriented technology to adopt data distribution and cybersecurity without doubts on being the first mover."
+        title="Condensation meets the security and technical requirements to start distributing sensitive data in an industrial context."
       />
 
       <div className="grid grid-cols-1 sm:grid-cols-2 px-8 pt-10 sm:px-28 sm:pt-16 sm:pb-10 gap-10 max-w-screen-xl">
-        <div className="max-w-md basis-120">
+        <div className="max-w-lg basis-120">
           <h3 className="text-xl sm:text-2xl font-bold text-transparent bg-clip-text bg-gradient-to-r from-blue-500 to-sky-500">
             A lightweight and open-source code base.
           </h3>
           <div className="pt-4 text-base">
-            You can use Condensation for commercial purpose for free, it is organised in lightweight
-            modules which can be audited by a single person. Moreover, it can be customized with new
-            implementations for specific purposes such as preferred encryption methods or
-            pre-defined conflict resolution rules.
+            Condensation can be used for commercial purpose without any fees. It is organized in
+            three lightweight modules that can be easily audited. Moreover, its implementation can
+            be customized for specific needs, such as preferred encryption algorithms or pre-defined
+            conflict resolution rules.
           </div>
         </div>
 
-        <div className="max-w-md basis-120">
+        <div className="max-w-lg basis-120">
           <h3 className="text-xl sm:text-2xl font-bold text-transparent bg-clip-text bg-gradient-to-r from-blue-500 to-sky-500">
-            Generic to build tailored solutions.
+            Generic enough to build tailored solutions.
           </h3>
           <div className="pt-4">
-            You can manage the complete lifecycle of data with Condensation, that’s why it is called
-            a data-system. It excels with synchronization performances in a fully distributed setup,
-            provides a single protocol to connect multiple devices with end-to-end data security,
-            and stores data without centralization constraints.
+            Condensation is a data-system, i.e able to manage a complete data lifecycle. It excels
```

Comment:
We explain the features here but its the purpose of the next section, here we should rather stick to the title, to say in more details we can build any kind of apps... messengers, collaborative documents, IoT, etc...